### ISSUE TITLE
If the font size of the shapes in the template file is different, the space is not saved successfully

### ISSUE THREAD

**Issue by 93910676wangd** [EPPlusSoftware/EPPlus#676]
There are multiple shapes in the template file. Please see the attachment  picture and the attachment "input.xlsm"
Some characters in each shape have the same font size, and some are inconsistent (including spaces)
It is found that the space between words will be deleted after the template is saved.
--------------------------------------------------
① in the picture is that the spaces have been deleted (because the fontsize of the spaces is inconsistent with the font)
② in the picture, if the font size in the graphic is the same (except for English letters), the space will be retained. This is no problem.
Although the font size of ③ in the picture is the same, because the characters are English letters,
After saving, the spaces between the text will be deleted.

①、③→✖

I think this is a bug. Please confirm.
If it is a bug, when will the problem be fixed?
![result](https://user-images.githubusercontent.com/19505572/176618636-23618db8-c62d-4588-b43b-b194b53d7e5b.png)

========================
"EPPlus" Version="5.8.6"
or
"EPPlus" Version="6.0.5"
========================
sample source code
~~~C#
            using ExcelPackage excelS = new ExcelPackage(new FileInfo(@"c:\input.xlsm"));
            var sheet04 = excelS.Workbook.Worksheets["4"];
            sheet04.Cells["P10"].Value = 10;
            excelS.SaveAs(new FileInfo(@"c:\output.xlsm"));
~~~
========================
test files：[testfile.zip](https://github.com/EPPlusSoftware/EPPlus/files/9017403/testfile.zip)

**Comment by JanKallman** [comment]
This looks like an issue when the drawing xml is loaded. EPPlus sets preseveWhitespaces=false...
https://github.com/EPPlusSoftware/EPPlus/blob/947bef1ec55c523bbda1f6f728e68a8f7efc06f9/src/EPPlus/Drawing/ExcelDrawings.cs#L79
Setting it to true solves the issue and keeps the whitespace as expected. I'll provide a fix shortly.

**Comment by 93910676wangd** [comment]
> This looks like an issue when the drawing xml is loaded. EPPlus sets preseveWhitespaces=false...
> 
> https://github.com/EPPlusSoftware/EPPlus/blob/947bef1ec55c523bbda1f6f728e68a8f7efc06f9/src/EPPlus/Drawing/ExcelDrawings.cs#L79
> 
> 
> Setting it to true solves the issue and keeps the whitespace as expected. I'll provide a fix shortly.

Thank you for your confirmation and look forward to the release of the new version

**Comment by JanKallman** [comment]
Fixed in version 6.0.6
